Update and simplify DocSum tutorials

alexsin368 opened the pull request:
- Remove steps to build images: just use docker compose to pull images instead
- Add steps for running with vLLM and TGI
- Update steps to validate microservices
- Update steps for port forwarding and changing BACKEND_SERVICE_IP for UI to work
- Fix grammar and improve wording

mkbhanda commented on tutorial/DocSum/deploy/gaudi.md:
```diff
@@ -229,40 +107,70 @@ Here are some sample messages if proxy environment variables are not set:
+
+In the first startup, this service will take more time to download, load, and warm up the model. After it is finished, the service will be ready.
+
+Try the command below to check whether the LLM serving is ready.
```
Perhaps you could add a sentence, "Note, in the below command, we are using the name of the image to check on its status. Just replace this name should you be running on different hardware or using another model server, to check whether the service is ready."

alexsin368 replied:
This should suffice: "Try the command below to check whether the LLM service is ready. It uses the name of the image to check the status."

mkbhanda commented on tutorial/DocSum/deploy/gaudi.md:
```diff
@@ -271,240 +179,276 @@ Here is the output:
 :sync: Video
 
-Video uploads are not supported through curl command, use the UI to upload it. You can pass base64 string of the video file as the value for message parameter as shown here :
+Video uploads are not supported through *curl* commands, so use the UI to upload it. It is possible to pass base64 strings of the video file as the value for the message parameter:
```
```suggestion
Video uploads are not supported through *curl* commands, so use the UI to upload it. It is possible to pass base64 encoded strings of the video file as the value for the message parameter:
```

mkbhanda commented on tutorial/DocSum/deploy/gaudi.md:
```diff
@@ -271,240 +179,276 @@ Here is the output:
+#### Megaservice with Long Context
 
-1. Auto : In this mode, we will check input token length, if it exceeds MAX_INPUT_TOKENS, the summary_type will automatically be set to refine mode, otherwise will be set to stuff mode.
+When dealing with longer context of the content to be summarized, different summarization strategies can be used such as *auto*, *stuff*, *truncate*, *map_reduce*, or *refine*. The best strategy is determined from various factors including model context size and input tokens.
```
```suggestion
When dealing with longer context of the content to be summarized, different summarization strategies can be used such as *auto*, *stuff*, *truncate*, *map_reduce*, or *refine*. The best strategy is determined from various factors including model context size limit and number of input tokens.
```

mkbhanda commented on tutorial/DocSum/deploy/xeon.md:
```diff
@@ -231,40 +106,71 @@ Here are some sample messages if proxy environment variables are not set:
+
+### vLLM or TGI Service
+
+In the first startup, this service will take more time to download, load, and warm up the model. After it is finished, the service will be ready.
```
```suggestion
On first startup, this service will take more time to download, load, and warm up the model. After it is finished, the service will be ready.
```

alexsin368 replied:
changed to "During the initial startup, this service will take a few minutes to download the model files and complete the warm-up process. Once this is finished, the service will be ready for use."

mkbhanda commented on tutorial/DocSum/deploy/xeon.md:
```diff
@@ -273,229 +179,276 @@ Here is the output:
+#### Megaservice with Long Context
 
-1. Auto : In this mode, we will check input token length, if it exceeds MAX_INPUT_TOKENS, summary_type will automatically be set to refine mode, otherwise will be set to stuff mode.
+When dealing with longer context of the content to be summarized, different summarization strategies can be used such as *auto*, *stuff*, *truncate*, *map_reduce*, or *refine*. The best strategy is determined from various factors including model context size and input tokens.
```
```suggestion
When dealing with long contexts, that is content longer than the model's context limit, for summarization, different summarization strategies can be used, such as *auto*, *stuff*, *truncate*, *map_reduce*, or *refine*. The best strategy is determined from various factors including model context size and input tokens.
```

alexsin368 replied:
changed to "When performing summarization with long contexts - content longer than the model's context limit - different summarization strategies can be used such as *auto*, *stuff*, *truncate*, *map_reduce*, or *refine*."

mkbhanda reviewed (commented):
@alexsin368 could you have introduced a tab for the HW? There is so much repetition here between Gaudi and CPU. The only differences would have been which compose file to use to start and stop services. I know it is a pain, but longer term less to maintain. Please consider.

alexsin368 commented:
@mkbhanda I've addressed all your comments and suggestions. You have a valid point about adding a tab for different hardware. However, for examples with multiple model serving options vLLM and TGI, we will end up with nested tabs which can make it not user friendly. A lot of logs are also different.

mkbhanda reviewed (approved):
LGTM
